README.md

```diff
@@ -215,6 +215,9 @@ Yes it is! This plugin comes with .po and .mo files. It is already translated in
 
 ## Changelog ##
 
+### 1.5.11.2 ###
```

Comment:
version should be 1.5.11.1

Reply:
1.5.11.1 has been given to phpstan fixes branch